public/app/features/dashboard-scene/embedding/EmbeddedDashboardScene.tsx

```diff
@@ -0,0 +1,115 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { CoreApp, DataQueryRequest, GrafanaTheme2 } from '@grafana/data';
+import { SceneObject, SceneObjectBase, SceneObjectState, SceneComponentProps } from '@grafana/scenes';
+import { DashboardLink } from '@grafana/schema';
+import { useStyles2 } from '@grafana/ui';
+import { DashboardMeta } from 'app/types';
+
+import { getClosestVizPanel, getPanelIdForVizPanel } from '../utils/utils';
+
+export interface EmbeddedDashboardSceneState extends SceneObjectState {
+  /** The title */
+  title: string;
+  /** Tags */
+  tags?: string[];
+  /** Links */
+  links?: DashboardLink[];
+  /** A uid when saved */
+  uid?: string;
+  /** Layout of panels */
+  body: SceneObject;
+  /** Fixed row at the top of the canvas with for example variables and time range controls */
+  controls?: SceneObject[];
+  /** meta flags */
+  meta: DashboardMeta;
+}
+
+export class EmbeddedDashboard extends SceneObjectBase<EmbeddedDashboardSceneState> {
```

Comment:
Nit, how about `ReadOnlyDashbaord`? 

Think this coul also be reusable for ie. snapshots page.... WDYT?

Reply:
@dprokop maybe, the snapshot is a bit different (it's a page and needs to interact with the page breadcrumbs etc). 

But I do think there is going to be similarities in how we build the DashboardScene , some things will only be added when !meta.isEmbedded or meta.isSnapshot etc